crates/autopilot/src/domain/competition/mod.rs

```diff
@@ -1,98 +1,16 @@
-use {
-    super::{auction, eth},
-    crate::domain,
-    std::collections::HashMap,
-};
+use super::eth;
 
-type SolutionId = u64;
+mod solution;
 
-#[derive(Debug)]
-pub struct Solution {
```

Comment:
The whole file (without any changes) is moved to domain::competition::solution.rs